main/src/ca/uwaterloo/flix/language/ast/Kind.scala

```diff
@@ -26,7 +26,7 @@ sealed trait Kind {
   /**
     * Constructs an arrow kind.
     */
-  def ->(that: Kind): Kind = Kind.Arrow(List(this), that)
+  def ->:(left: Kind): Kind = Kind.Arrow(List(left), this)
```

Comment:
I did a double take when I saw the argument order.. But after staring at it for a while, I guess it makes sense!